tests/fixtures/sessions/sess.json

```diff
@@ -0,0 +1 @@
+{"tab_scoped":false,"quickfix":[],"global":{"height":66,"options":{"cmdheight":1},"width":254,"cwd":"\/home\/gcloud\/resession.nvim"},"tabs":[{"wins":["leaf",{"bufname":"\/home\/gcloud\/resession.nvim\/tests\/fixtures\/files\/file1","cursor":[1,0],"current":false,"width":254,"height":65,"options":{"diff":false,"winfixwidth":false,"winfixheight":false,"scrollbind":false,"previewwindow":false}}]},{"wins":["leaf",{"bufname":"\/home\/gcloud\/resession.nvim\/tests\/fixtures\/files\/file2","cursor":[1,0],"current":true,"width":254,"height":64,"options":{"diff":false,"winfixwidth":false,"winfixheight":false,"scrollbind":false,"previewwindow":false}}]}],"buffers":[{"name":"\/home\/gcloud\/resession.nvim\/tests\/fixtures\/files\/file1","last_pos":[1,0],"options":{"modifiable":true,"bufhidden":"","filetype":"","buflisted":true,"readonly":false,"binary":false},"loaded":true},{"name":"","last_pos":[1,0],"options":{"modifiable":true,"bufhidden":"hide","filetype":"","buflisted":false,"readonly":false,"binary":false},"loaded":true},{"name":"","last_pos":[1,0],"options":{"modifiable":true,"bufhidden":"hide","filetype":"cmp_menu","buflisted":false,"readonly":false,"binary":false},"loaded":true},{"name":"\/home\/gcloud\/resession.nvim\/tests\/fixtures\/files\/file2","last_pos":[1,0],"options":{"modifiable":true,"bufhidden":"","filetype":"","buflisted":true,"readonly":false,"binary":false},"loaded":true}]}
```

Comment:
It would be nice to be able to specify the session in object format in the `_spec.lua` file rather than having to create a JSON file as part of the fixtures. But that would involve decoupling some parts of the `resession.load` function, which expects a file name rather than the session data itself, so I thought it would be best left to a future PR.